ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/BlockPropagationManager.java

```diff
@@ -63,6 +63,7 @@
   private final EthContext ethContext;
   private final SyncState syncState;
   private final LabelledMetric<OperationTimer> ethTasksTimer;
+  private BlockBroadcaster<C> blockBroadcaster;
```

Comment:
This field is always overwritten prior to use. Consider making it a variable instead.